### DISCUSSION TITLE
Is it possible to change image resolution

### DISCUSSION THREAD

**Discussion by C-Coretex** [firecmsco/firecms#237] [Q&A] [answered]
I'm using FireCMS to add images to the storage. Then phone application downloads the images.
Is it possible to change the resolution of the images, so the size of the images wouldn't be so big? 

![image](https://user-images.githubusercontent.com/44605873/154795119-02302332-f6e6-4110-8a75-4b6979237a41.png)

**Comment by uzproblue** [comment]
You can use libraries like "compressorjs" to compress the image before uploading it to the cloud.
Also, you can check out this article: https://medium.com/front-end-weekly/image-compression-in-reactjs-a07ec0066b24.

**Comment by fgatti675** [comment, marked as answer]
Hi @C-Coretex 
It is not currently possible to modify the uploaded files, though we might add it in the future.
We have the same issue and we solve it using the resize image extension of Firebase.

**Reply by C-Coretex** [reply]
Ok. The extension requires a billing plan, so it's not the best solution for me.
Look forward to getting this feature in FireCMS
